sites/svelte.dev/src/routes/faq/+page.js

```diff
@@ -0,0 +1,5 @@
+import { redirect } from '@sveltejs/kit';
```

Comment:
Why not add this in +page.server.js with prerendering?

Reply:
my original thinking was 'if it's a client-side `load`, redirecting on the client will be instant' but that doesn't really make sense given that there are no longer any links you could click to get a client-side redirect. will change it to a `+server.js`